demos/shadow/shadow_demo_main/shadow_demo_main.c

```diff
@@ -69,6 +70,18 @@
 #include "shadow_demo_helpers.h"
 
 
+/**
+ * @brief Predefined shadow name.
+ *
+ * Defaults to un-named "Classic" shadow. Change to a custom string to use a named shadow.
+ */
+#define SHADOW_NAME           SHADOW_NAME_CLASSIC
```

Comment:
I think this would benefit from being demo user configurable.
```suggestion
#ifndef SHADOW_NAME
    #define SHADOW_NAME           SHADOW_NAME_CLASSIC
#endif
```

Reply:
Moved it to demo_config.h and made it similar to how THING_NAME is handled.

Reply:
I am fine with this. Thanks!